plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -67,10 +67,10 @@ func defaultPriorities() util.StringSet {
 		factory.RegisterPriorityFunction("BalancedResourceAllocation", priorities.BalancedResourceAllocation, 1),
 		// spreads pods by minimizing the number of pods (belonging to the same service) on the same minion.
 		factory.RegisterPriorityConfigFactory(
-			"ServiceSpreadingPriority",
+			"SelectorSpreadPriority",
```

Comment:
1.0.0 shipped with `ServiceSpreadingPriority` (https://github.com/GoogleCloudPlatform/kubernetes/blob/release-1.0/plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go#L70). This rename breaks backwards compatibility with serialized scheduler config files. I'll open an issue to revert the name